bayesflow.utils.validators.ShapeError
=====================================

.. currentmodule:: bayesflow.utils.validators

.. autoexception:: ShapeError